app/forms/prior_authority/steps/check_answers_form.rb

```diff
@@ -13,15 +13,15 @@ class CheckAnswersForm < ::Steps::BaseFormObject
       def save!; end
 
       def persist!
+        updated = false
         application.with_lock do
-          update_application
+          updated = update_application
         end
+        SubmitToAppStore.perform_later(submission: application) if updated
+
+        true
```

Comment:
Is this return value used? Wouldn't returning `updated` be more appropriate?

Reply:
You're absolutely correct. Now fixed.